src/components/Router/StepComponentMap.js

```diff
@@ -44,15 +45,22 @@ const captureStepsComponents = (documentType, mobileFlow, steps, deviceHasCamera
   const complete = mobileFlow ? [ClientSuccess] : [Complete]
   return {
     welcome: () => [Welcome],
-    face: () => shouldUseVideo(steps) ?
-        getRequiredVideoSteps(deviceHasCameraSupport, mobileFlow) :
-        [SelfieCapture, SelfieConfirm],
+    face: () => getFaceSteps(steps, deviceHasCameraSupport, mobileFlow),
     document: () => createIdentityDocumentComponents(documentType, hasPreselectedDocument(steps)),
     poa: () => [PoAIntro, SelectPoADocument, PoAGuidance, PoACapture, DocumentFrontConfirm],
     complete: () => complete
   }
 }
 
+const getFaceSteps = (steps, deviceHasCameraSupport, mobileFlow) => {
+  const faceStep = steps.filter(step => step.type === "face")[0]
+  const useWebcam = faceStep.options.useWebcam === undefined ? true : faceStep.options.useWebcam
+  console.log('useWebcam:', useWebcam);
```

Comment:
Think you forgot to remove this `console.log`.

Also looks like we can't quite move away from needing to have a check for `useWebcam` property for dev/UI test use still.

Reply:
removed

Reply:
fixed